cvr_osc_lib/osc_messages_data.py

```diff
@@ -1,10 +1,18 @@
+"""Module to define the OSC messages data classes."""
+# Disable all the UPPER_CASE violations detections in this file as
+#  there are already too many scripts using this convention,
+#  to interface with this library and it would cause too much
+#  inconvenience for others if the names were changed.
+# pylint: disable=invalid-name
+
```

Comment:
I'm not using pylint, this project is using `flake8`
It allows Lower Case

Reply:
fixed in [86ed35e](https://github.com/kafeijao/cvr_osc_lib_py/pull/3/commits/86ed35e2105beca357a6a68722b632304fced53e)